qemu/tests/numa_prealloc_threads.py

```diff
@@ -0,0 +1,66 @@
+import re
+
+from virttest import env_process
+from virttest import error_context
+from virttest.qemu_monitor import QMPCmdError
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    numa_prealloc_threads test
+    1) Boot a guest with thread-context and cpu-affinity option
+    2) Obtain the thread-id
+    3) Check the affinity obtained from QEMU is correct
+    4) With sandbox enabled, try to change the cpu-affinity
+       and handle the error
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    test.log.info("Check host CPUs number")
+    host_cpus = int(process.system_output("nproc"))
```

Comment:
from avocado.utils import cpu
```suggestion
    host_cpus = int(cpu.online_count())
```